packages/components/src/components/date-picker/date-picker-calendar.tsx

```diff
@@ -1,13 +1,28 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { CSSTransition } from 'react-transition-group';
 import Calendar from '../calendar';
 import { useBlockScroll } from '../../hooks';
 
+type TDatePickerCalendar = {
+    value: string | null;
+    alignment?: string;
+    is_datepicker_visible: boolean;
+    calendar_el_ref: React.RefObject<HTMLElement>;
+    parent_ref: React.RefObject<HTMLElement>;
+    portal_id: string;
+    style: Record<string, string | number>;
+    placement: string;
+    onHover?: (Arg1: string) => void;
+    onSelect: (Arg1: string, Arg2: boolean) => void;
+};
```

Comment:
```suggestion
type TDatePickerCalendar = {
    value: string | null;
    alignment?: string;
    is_datepicker_visible: boolean;
    calendar_el_ref: React.RefObject<HTMLElement>;
    parent_ref: React.RefObject<HTMLElement>;
    portal_id: string;
    style: Record<string, string | number>;
    placement: string;
    onHover?: (Arg1: string) => void;
    onSelect: (Arg1: string, Arg2: boolean) => void;
};
```

Import types by making a type file and put all the types in that file.
Its more manageable that way.
Just a suggestion